frontend/packages/rhoas-plugin/src/components/service-list/ServiceListPage.tsx

```diff
@@ -68,32 +91,18 @@ const ServiceListPage: React.FC = () => {
       setSubmitting(false);
     } catch (error) {
       deleteKafkaConnection(name, currentNamespace);
-      setKafkaCreateError(error);
-      setSubmitting(false);
-    }
-  }, [currentNamespace, remoteKafkaInstances, selectedKafka]);
 
-  if (kafkaCreateError) {
-    return (
-      <ServicesErrorState
-        title={t('rhoas-plugin~Failed to create connection')}
-        message={kafkaCreateError + t('rhoas-plugin~Please try again')}
-        actionLabel={t('rhoas-plugin~Go back to Services Catalog')}
-      />
-    );
-  }
+      const connectionErrorObj = {
+        title: t('rhoas-plugin~Failed to create connection'),
+        message: error + t('rhoas-plugin~Please try again'),
```

Comment:
@wtrocki should we remove `Please try again` here for the moment, since its contradicting with the action of `Go back to services catalog` ?